### ISSUE TITLE
Converting pc to mobile input in pygame

### ISSUE THREAD

**Issue by YashDhankhar** [python/python-docs-zh-cn#158]
I made a simple floppy bird game using PYGAME.
So I thought that I should try this game on my phone.
So I am having an input of the game which is "When I'll click spacebar the bird should jump"
for uploading it to mobile    
How will I convert the spacebar function to mobile tap function.
![error](https://user-images.githubusercontent.com/75427668/101661028-cd96f200-3a6d-11eb-8508-67027670928f.jpg)

so basically I want an input including tap function for the bird to fly on mobile phone
by keyboard I can do that by pressing space or up arrow.

**Comment by zhsj** [comment]
Hi, this place is issue tracker for Chinese translation of Python documents, aka https://docs.python.org/zh-cn. If you want to discuss generic Python topics, it's better to check one of the communities in https://www.python.org/community/.
